2dcontext/imagebitmap/createImageBitmap-serializable.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>createImageBitmap serialize test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/media.js"></script>
+<script src="/common/namespaces.js"></script>
```

Comment:
are those needed?

Reply:
Not sure, I derived this test from createImageBitmap-transfer.html

Reply:
Sure. Please remove the ones that are not needed.

Reply:
Only media.js isn't needed I think, I've removed it - namespaces isn't strictly needed, but is a dependency of common.sub.js below, and I suppose a linter might fail if it isn't included(?)